subsys-srr.typ

```diff
@@ -0,0 +1,67 @@
+#import "@preview/bytefield:0.0.5": *
+#import "lib/pugh.typ": *
+
+= Short Range Radio (SRR)
+
+The Short Range Radio (SRR) is the system used to let colonists speak to each other on the colony. The goal of this system to enable colonists to have reliable communication amongst themselves in any conditions at long-distance so that missions can be conducted on the surface without being directly next to each other. The focus is reliability, as being able to talk during missions is as vital as a life support system. It will utilize commonly used radio equipment that is slightly modified to better fit the use-case of an interplanetary mission.
+== Architecture
+
+The overview of the SRR is similar to a commonly found short range radio communication system on Earth. Each colonist will have a walkie-talkie device equipped with an omni-directional transceiver in the form of a whip antenna. This will allow them to communicate by sending voice messages up to 6 miles away. However, we also plan to send the colonists with a separate transceiver that can be attached to the tallest point on the base, which can allow voice messages to be sent to other colonists at a range of 15-20 miles. The biggest limiting factor is the height of the transceivers, but by using the UHF (ultra high frequency) band as the communication medium, we hope to keep the range as far as physically possible.
+=== Alternatives Considered
+
+For short range communication, alternatives include physical conversation (no technology required), or creating an entire novel method of communication.
+
+The alternatives have huge issues, though. Physical conversation is implausible as there needs to be more distance than immediate arms-length to allow for conversation for missions and communication across the colony. A novel method may have a better upside of range, but to create an entire novel SRR system would greatly risk the reliability necessary for the system. It is also extremely implausible to create an entire new methodology within an already high-risk mission.
+
+As a result, we did not believe a Pugh matrix was necessary to showcase that, of all possibilities, using established radio technology is most definitely the way to go.
+
+
+== Handheld Radio Devices
+
+The most fundamental part of this system is the handheld radio device that each colonist will have. It is, by all intents and purposes, a walkie-talkie. However, it will be specifically resilient to impact, dusty conditions, and radiation. It will be encased in a plastic-covered lead plating that ensures that it can endure the conditions of the Martian surface. 
```

Comment:
I don't think this would work too well for people in EVA suits. Those are air tight, so if the radio is in their hand, how will they hear the radio? Or be able to talk into it?

Reply:
I gave this some thought, but I considered that people in the base camp would still need handhelds regardless AND Mars still has an atmosphere where sound can permeate for about ~100m @ 100dB, so hearing and speaking shouldn't be a problem at all. I figured it might be better so the suit wouldn't have to be worn indoors to still communicate with colonists on mission.

Reply:
There's no reason to have only one type of radio. Handhelds could be supplementary to integrated radios. Mars atmosphere is not breathable and Martian soil is toxic. Would the suits just have mics and speakers on them?

Reply:
My point was that there doesn't need to be a mic/speaker system on the suit. It may be air-tight, but the atmosphere can carry sound waves on Mars. They will speak in their suit, which will permeate through the air in their suit, through the barrier, and into the Martian atmosphere. It isn't a vacuum.

Reply:
I think it would add a layer of complexity to incorporate an integrated suit-speaker system which goes against the high reliability requirement and it would also need a giant floppy wire antenna to be effective as per the 6 miles I quoted.

Reply:
We aren't in control of the suit design. If you put a bucket on your head, your voice gets muffled. Now imagine the bucket completely surrounds you, and there's also [multiple layers of different materials, including fabric](https://www.nasa.gov/centers-and-facilities/johnson/nasas-perseverance-rover-will-carry-first-spacesuit-materials-to-mars/) too. 

Now, if we just give them handheld walkie talkies with puny speakers and microphones:
1. The colonists are not going to be able to hear the radio very well, or speak into the radio clearly.
2. The handheld radios occupy a hand, and that's one less free hand to do things with.
3. Handheld radios usually have little holes in them for the microphone and speakers so that they aren't muffled. That's a dust trap.

Why wouldn't we just put radios in the suits?